### PR TITLE
dpg, use setPlaybackSyncPollerPollInterval in test

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
fix https://github.com/Azure/autorest.java/issues/2602

### CODE REVIEW

File: javagen/src/main/java/com/azure/autorest/template/example/ProtocolExampleWriter.java [hunk 1]
```diff
@@ -222,12 +224,14 @@ public ProtocolExampleWriter(ProtocolExample protocolExample) {
...
-            methodBlock.line(String.format(
-                    "%1$s response = %2$s.%3$s(%4$s);",
-                    method.getReturnValue().getType(),
+            String methodCall = String.format("%s.%s(%s)",
                     clientVarName,
                     method.getName(),
-                    String.join(", ", params)));
+                    String.join(", ", params));
+            if (isTestCode && (method.getType() == ClientMethodType.LongRunningBeginSync || method.getType() == ClientMethodType.LongRunningBeginAsync)) {
+                methodCall = "setPlaybackSyncPollerPollInterval(" + methodCall + ")";
```
Comment (XiaofeiCao) [inline javagen/src/main/java/com/azure/autorest/template/example/ProtocolExampleWriter.java:232]: Should we use `setPlaybackPollerFluxPollInterval` for `LongRunningBeginAsync`?

Comment (weidongxu-microsoft) [reply javagen/src/main/java/com/azure/autorest/template/example/ProtocolExampleWriter.java:232]: Yes, the method has overload for PollerFlux

Comment (XiaofeiCao) [reply javagen/src/main/java/com/azure/autorest/template/example/ProtocolExampleWriter.java:232]: Their names seem to be different? `setPlaybackPollerFluxPollInterval` vs `setPlaybackSyncPollerPollInterval`
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core-test/src/main/java/com/azure/core/test/TestBase.java#L351-L378

Comment (weidongxu-microsoft) [reply javagen/src/main/java/com/azure/autorest/template/example/ProtocolExampleWriter.java:232]: Ah, I see. It is not an overload. Will fix.

**Review by XiaofeiCao — APPROVED** [review]

**Pull request merged**